CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+# Changelog
+
+This project follows the semantic versioning without the leading v in the tags.
+
+Every Message starts with one of the following words:
+
+ - `Add` for new functionality
+ - `Change` for change in behavior
+ - `Break` for breaking-change
+ - `Fix`
+ - `Switch` internal change
+ - `Remove` functionality
+
+## [Unreleased]
+
+ - Add fetch pictures from all webcams
+ - Add CHANGELOG.md
+
+## [0.2.6] - 2015-11-25
+
+ - Change logging to env_logger crate
+ - Change response behavior of bot in groups
+ - Add bot logo
+
+## [0.2.5] - 2015-11-23
+
+ - Switch to types from spaceapi crate
+
+## [0.2.4] - 2015-11-18
+
+ - Change /crowd to /status
+ - Switch to grammar parsing
+ - Add more grammar rules
+
+## [0.2.3] - 2015-11-16
+
+ - Add travis ci integration
+ - Add /version
+ - Add /help
+
+## [0.2.2] - 2015-11-16
+
+ - Fix telegram-bot crate
+ - Add exponential backoff
+
+## [0.2.1] - 2015-11-14
+
+ - Add grammar for user input
+
+## [0.2.0] - 2015-11-12
+
+ - Add /crowd to answer people_now_present
+
+## [0.1.0] - 2015-11-11
+
+ - Initial Release
+
+[Unreleased]: https://github.com/coredump-ch/coredumpbot/compare/v0.2.6...HEAD
```

Comment:
Tags use what you define. But I'm still not so sure about what actually should be used. There where quite some use cases where it was really useful for me that tags were using a `v`prefix.

* here it seems clear: http://semver.org/spec/v1.0.0.html#tagging-specification-semvertag
* but: https://github.com/mojombo/semver/issues/1
* and: https://github.com/mojombo/semver/issues/204

So version 2 of semver doesn't specify it anymore.